tsup.config.ts

```diff
@@ -6,7 +6,7 @@ export const tsup: Options = {
   target: 'esnext',
   clean: true,
   dts: true,
-  entry: ['src/index.tsx', 'src/flows'],
+  entry: ['src/index.tsx', 'src/flows/*.tsx'],
```

Comment:
```suggestion
  entry: ['src/index.tsx', 'src/flows/*.ts*'],
```
I think it's better if we support both ts and tsx.